### ISSUE TITLE
Allow windows to be resized

### ISSUE THREAD

**Issue by emjayteeou** [antlr/antlr4-lab#66]
Small point but it would be nice if one could change the width of the left hand window where the grammar is defined.
There's a scroll bar at the bottom but would be good to be able to see longer definitions / comments with reasonable font size.

**Comment by kzeitz** [comment]
Yes agree!
If you generate a long error, the lexer/parser editor section get's small and you can't resize them at all.

**Comment by parrt** [comment]
I totally agree but my limited CSS skills have defeated me! haha. Would love some help here if you can figure out how to do that. It took me forever but I finally figured out how to resize the editor windows.

**Comment by emjayteeou** [comment]
Ah, it's always the "nuts & bolts" that are a problem. I'd love to help but my CSS skills are even worse than my other programming skills....
If there's someone out there who can do it (I'm sure its not hard when you know how), it would be a real help!

Thanks
